src/sql/store.py

```diff
@@ -120,13 +120,15 @@ def __str__(self) -> str:
         ` (backtick)
         """
         with_clause_template = Template(
-            """WITH{% for name in with_ %} {{name}} AS ({{rts(saved[name]._query)}})\
-{{ "," if not loop.last }}{% endfor %}{{query}}"""
+            """WITH{% for name in with_ %} {{name}} AS (\
+{{'\n\t' + rts(saved[name]._query) + '\n'}}\
+){{ "," if not loop.last }}{% endfor %}{{'\n' + query}}"""
         )
 
         with_clause_template_backtick = Template(
-            """WITH{% for name in with_ %} `{{name}}` AS ({{rts(saved[name]._query)}})\
-{{ "," if not loop.last }}{% endfor %}{{query}}"""
+            """WITH{% for name in with_ %} `{{name}}` AS (\
+{{'\n\t' + rts(saved[name]._query) + '\n'}}\
```

Comment:
why did we add a newline here?

Reply:
I was printing out the rendered sql to see how the code was working. I added a newline and tabs here because it made it much more easier to view the opening parentheses and their corresponding closing parentheses. This also formats the query in case the user uses the `sqlcmd snippets some_snippet` command.